performance/config.py

```diff
@@ -29,18 +27,13 @@ class Config:
 
     def __init__(self):
         # TODO: grab these from ENV variables
-        self.PIWIK_AUTH_TOKEN = self._get_piwik_token()
+        self.PIWIK_AUTH_TOKEN = os.getenv('PIWIK_API_TOKEN')
         self.rp_information = {rp['rp_name']: rp for rp in self._load_json_configuration('rp_information')}
         self.rp_mapping = self._load_json_configuration('rp_mapping')
         self._validate_rp_information()
 
-    def _get_piwik_token(self):
```

Comment:
So is not the problem with doing this that PA's now have to maintain their PIWIK token in two places? It seems like ideally we'd be able to look in the environment, but still have using the data pipeline config as a fallback, perhaps? Maybe worth a discussion.